### PR TITLE
updated ticket info

### PR DESCRIPTION
**Pull request by Paolaufer**

### CODE REVIEW

File: tutorials/appstudio-cap-deploy-app/appstudio-cap-deploy-app.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ author_profile: https://github.com/Paolaufer
 
 ## Prerequisites
  - You have an SAP HANA service (SAP HANA as a Service or SAP HANA Cloud) available in your space.
+    To create a new instance in your trial account:
+    1. Go to your space in the SAP Cloud Platform cockpit.
+    2. Go to **Service Marketplace**
```
Comment (MichaelCzcz) [inline tutorials/appstudio-cap-deploy-app/appstudio-cap-deploy-app.md:17]:
```suggestion
    2. Go to **Service Marketplace**.
```

Comment (MichaelCzcz) [inline tutorials/appstudio-cap-deploy-app/appstudio-cap-deploy-app.md:17]:
```suggestion
    2. Go to **Service Marketplace**.
```

File: tutorials/appstudio-cap-deploy-app/appstudio-cap-deploy-app.md [hunk 2]
```diff
@@ -12,6 +12,12 @@ author_profile: https://github.com/Paolaufer
...
+    3. Select **SAP HANA Schemas & HDI Containers (SAP HANA Trial)**
```
Comment (MichaelCzcz) [inline tutorials/appstudio-cap-deploy-app/appstudio-cap-deploy-app.md:18]:
```suggestion
    3. Select **SAP HANA Schemas & HDI Containers (SAP HANA Trial)**.
```

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Just some minor things to keep those lines consistent with the other ones.

**Comment by Paolaufer** [comment]
added your comments

**Pull request merged**